stam.py

```diff
@@ -0,0 +1,49 @@
+import sys
+import os
```

Comment:
I added a style guide to the git repo. Check what it says about imports. (I have a bunch of comments along those lines. Other than that pretty much everything is great)